mapping/stations.py

```diff
@@ -57,3 +59,78 @@ def map_stations_bna(row):
     new_station.coordinates = coordinates
     new_station.date_created = row["Inbetriebnahmedatum"].strftime("%Y-%m-%d"),
     return new_station
+
+
+def map_stations_ocm(row):
+    lat = check_coordinates(row["AddressInfo.Latitude"])
+    long = check_coordinates(row["AddressInfo.Longitude"])
+    operator: Optional[str] = (
+        row["Title"]
+    )
+    new_station = Station()
+    new_station.source_id = lat_long_hash(lat, long)
+    new_station.operator = operator
+    new_station.data_source = "OCM"
+    coordinates = Point(
+        float(lat),
+        float(long)
+    ).wkt
+    new_station.coordinates = coordinates
+    new_station.date_created = row["DateCreated"]
+    return new_station
+
+
+def map_address_ocm(row, station_id):
+    try:
+        country: Optional[Dict] = row["Title_y"]
+    except:
+        country = None
+    postcode: Optional[str] = (
+        row["AddressInfo.Postcode"]
+    )
+    postcode = (
+        "".join([s for s in postcode if s.isdigit()])
+        if postcode is not None
+        else ""
+    )
+    town: Optional[str] = (
+        row["AddressInfo.Town"]
+    )
+    if town is None:
+        town = ""
```

Comment:
Why is town set to "" if it's None anyway? Later on when there is the validity check with len(town)<2 we set town=None and it seems ok to use the None value in further processing